README.md

```diff
@@ -1,46 +1,118 @@
-# WalletLink
+# Coinbase Wallet SDK
 
-WalletLink is an open protocol that lets users connect their mobile wallets to
-your DApp.
+Coinbase Wallet SDK (formerly WalletLink) is an open protocol that works with 
+your dapps on both mobile web (for iOS and Android) and desktop: 
 
-With WalletLink, users can use your application in any desktop browser without
-installing an extension, and the secure tunnel established between your app and
-the mobile wallet with end-to-end encryption utilizing client-generated keys
-keeps all user activity private.
+- **Mobile**: Users can connect to your mobile app through a deeplink to the 
```

Comment:
`mobile app` is confusing wording to me. I'd suggest using the word dapp instead. the dapps users interact with through the dapp browser are designed for web, they are not specifically _mobile apps_

Reply:
Done